Fix a couple transcription errors and italicize a couple foreign-language expressions

MagisterAdamus opened the pull request:
Some notes:

1. The word `cientipedoro` is not a real Spanish word, even though it is supposed to sound like Spanish. It's a made-up word used only in this work. [In the scans](https://archive.org/details/bub_gb_RvEOAAAAIAAJ/page/n76/mode/1up), there is an explanatory footnote that explains the origins of this made-up insect and how it arrived in Spain. This note is missing currently from the SE edition of this work.
2. The phrase `in fine` should be italicized and marked up as Latin according to [SEMoS §8.2.9.9](https://standardebooks.org/manual/1.8.4/8-typography#8.2.9.9).

I've only read about a quarter of this novel so far. There seem to be extensive differences in punctuation between the scans and the transcription, and I'm not quite sure why.

MagisterAdamus committed:
Italicize and mark up 'diavolo' as Italian

MagisterAdamus committed:
Fix transcription error

IA scan reference: https://archive.org/details/bub_gb_RvEOAAAAIAAJ/page/n58/mode/1up

MagisterAdamus committed:
Fix transcription error

IA scan reference: https://archive.org/details/bub_gb_RvEOAAAAIAAJ/page/n84/mode/1up

MagisterAdamus committed:
Remove italics and Spanish-language semantics from made-up word 'cientipedoro'

acabal commented:
Are you sure `in fine` is Latin? M-W doesn't mention it: https://www.merriam-webster.com/dictionary/fine#in-fine It's not italicized anywhere else in the corpus and occurs fairly often.

It's possible the transcription was from a different edition, which is why punctuation may differ.

It would be good to add the missing endnotes. Can you do that?

MagisterAdamus commented:
> Are you sure `in fine` is Latin? M-W doesn't mention it: https://www.merriam-webster.com/dictionary/fine#in-fine It's not italicized anywhere else in the corpus and occurs fairly often.

Yes, I'm 100% sure that it's Latin. (I took one semester of Latin during my undergraduate degree.) In Latin, most prepositions, including the Latin preposition `in` (which means 'in' here), take the ablative case of the following noun. In other words, a prepositional phrase in Latin is typically composed of a head preposition followed by a "noun phrase," with the noun phrase's head noun in the ablative case. The Latin word `fīne` is the ablative form of the third-declension masculine noun `fīnis, -is`, meaning '(an) end'. So `in fīne` is Latin for 'in the end'; there were no articles in Classical Latin.

**Edit:** The _Oxford English Dictionary_ [states](https://www.oed.com/dictionary/fine_n1?tab=meaning_and_use#4190207) that in later uses of this phrase, it mostly occurs in the form `in fine`, modeled on Classical Latin `in fīnē`. But it also appears as `in the fine` and `at fine` as variants of the same expression.

> It's possible the transcription was from a different edition, which is why punctuation may differ.

Yes, that must be it. But it would be a lot of work to update the punctuation to match the scans.
 
> It would be good to add the missing endnotes. Can you do that?

Yes, I can do this. If I do this, should I also move the translations of the epigraphs into the endnotes? That's what was done for William Wycherley's _The Country Wife_. This would affect the opening epigraph for the whole work from Horace and the one in Italian from Tasso, which I believe is for Chapter 2.

acabal commented:
> Yes, I'm 100% sure that it's Latin. 

Let's leave it without italics. I'm not convinced that it's understood to be a Latin phrase in typical English parlance. There is exactly one instance of `in fine` in italics in the entire corpus, even though `in fine` appears with regularity. This is in contrast with many other common Latinisms which were frequently italicized even if they wouldn't be today.

> But it would be a lot of work to update the punctuation to match the scans.

You don't have to do that. It's common for this to occur, we typically leave the transcription as-is.

> Should I also move the translations of the epigraphs into the endnotes?

Sure.

MagisterAdamus committed:
Enclose name of psalm in quotation marks and add semantics to it

MagisterAdamus committed:
Remove hyphens from "ghostship"

The '-ship' morpheme here is the same '-ship' suffix as in 'Lordship' or 'Kingship'. Hyphenating it makes it look like a compound, which it isn't. It was unhyphenated in the source, too: https://archive.org/details/bub_gb_RvEOAAAAIAAJ/page/n140/mode/1up

MagisterAdamus commented:
> > Yes, I'm 100% sure that it's Latin.
> 
> Let's leave it without italics. I'm not convinced that it's understood to be a Latin phrase in typical English parlance. There is exactly one instance of `in fine` in italics in the entire corpus, even though `in fine` appears with regularity. This is in contrast with many other common Latinisms which were frequently italicized even if they wouldn't be today.

Okay! I've added a commit now to revert it, since I don't know how to do a rebase.

> > But it would be a lot of work to update the punctuation to match the scans.
> 
> You don't have to do that. It's common for this to occur, we typically leave the transcription as-is.

Okay!

> > Should I also move the translations of the epigraphs into the endnotes?
> 
> Sure.

I'll do this a little bit later.

I've pushed a couple more commits too now. `De Profundis` is apparently a traditional Catholic prayer based on Psalm 130, so I believe that it should be enclosed in quotation marks as per [SEMoS §8.2.11.3](https://standardebooks.org/manual/1.8.4/single-page#8.2.11.3). The word `ghostship` was hyphenated in the current SE release, but it shouldn't be; the -`ship` morpheme is a suffix, not a root, so the word shouldn't be hyphenated (it's not a compound). It was unhyphenated in the page scans, too. It's intended as a term of address or reference for a ghost, like `Lordship` or `Kingship`.

acabal commented:
> Okay! I've added a commit now to revert it, since I don't know how to do a rebase.

You should rebase it out. Deleting a commit with interactive rebase is easy: https://alchemists.io/articles/git_rebase_drop

Then force push to this PR

MagisterAdamus commented:
> > Okay! I've added a commit now to revert it, since I don't know how to do a rebase.
> 
> You should rebase it out. Deleting a commit with interactive rebase is easy: https://alchemists.io/articles/git_rebase_drop
> 
> Then force push to this PR

I can't seem to get it to work for some reason. (When I run `git rebase --interactive`, even after setting the upstream branch to track as `origin/various-changes`, there are no commits shown when my text editor opens up.)

One other question: Should moving the translations of the epigraphs be an [Editorial] change?

MagisterAdamus commented:
Hmm, I managed to get the rebase to work! I had to execute `git rebase --interactive master`. So I've removed the commit italicizing `in fine` and the commit reverting it now.

MagisterAdamus committed:
Fix typo

IA scan reference: https://archive.org/details/bub_gb_RvEOAAAAIAAJ/page/n74/mode/1up

MagisterAdamus committed:
Ch. 3: Restore missing word "to"

IA scan reference: https://archive.org/details/bub_gb_RvEOAAAAIAAJ/page/n103/mode/1up

MagisterAdamus committed:
Restore British spelling of "apprised"

This word-form occurs elsewhere as "apprised" in this SE release. IA scan reference: https://archive.org/details/bub_gb_RvEOAAAAIAAJ/page/n109/mode/1up

acabal commented:
Thanks!
